root/edit/search_macros.tt

```diff
@@ -424,6 +432,7 @@
                    [ 'vote_count', l('Vote tally') ],
                    [ 'edit_note_author', l('Edit Note Authors'), {requires_login => 1} ],
                    [ 'edit_note_content', l('Edit Note Content'), {requires_login => 1} ],
+                   [ 'edit_subscription', l('Edit'), {requires_login => 1} ],
```

Comment:
Just “Edit” in an edit search form doesn’t give any clue what this condition is about.

```suggestion
                   [ 'edit_subscription', l('Subscribed entity'), {requires_login => 1} ],
```

Reply:
I'm not sure about the wording but you're right about the 'Edit' name being confusing.

Reply:
It would be even clearer indeed.